front/lib/llm/index.ts

```diff
@@ -0,0 +1,26 @@
+import type { LLM } from "@app/lib/llm/llm";
+import type { AgentReasoningEffort } from "@app/types";
+import type { ModelConfigurationType } from "@app/types/assistant/models/types";
+import type { Result } from "@app/types/shared/result";
+import { Err, Ok } from "@app/types/shared/result";
+
+export function getLLM({
+  model,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  temperature,
```

Comment:
`temperature: _temperature`

The rule ignores the variables prefixed by "_", this pattern allows to see quickly if the variable is used or not.

Often used also when you want to get only the 2nd argument of a callback:
```
onSucess: (_, foo) => bar(foo)
```